### ISSUE TITLE
Value For Field renaming but file didn't rename

### ISSUE THREAD

**Issue by iqbal-ardiansyah22** [josegonzalez/cakephp-upload#372]
Hello,

I'm trying to renaming my file upload. but not working please should i mannualy renaming my file?

```php
 $this->addBehavior('Josegonzalez/Upload.Upload', [
            'image' => [
                'path' => 'webroot{DS}img{DS}{model}{DS}{field}{DS}',
                'fields' => [
                    'dir' => 'image_dir', // defaults to `dir`
                ],
                'nameCallback' => function($data,$settings){
                    $newName = uniqid().str_replace(" ", "_", $data['name']);
                    return $newName;
                }
            ],
        ]);
```

**Comment by jorisvaesen** [comment]
What version of the plugin are you using?
NameCallback got fixed in 3.2.0.

**Comment by jorisvaesen** [comment]
@iqbal-ardiansyah22 _bump_
